themes-default/slim/src/components/display-show.vue

```diff
@@ -89,6 +91,31 @@ export default {
             const id = show.id[show.indexer];
             const indexerUrl = indexerConfig[show.indexer].showUrl;
             return `${indexerUrl}${id}`;
+        },
+        statuses() {
+            const { statuses, config } = this.$store.state;
```

Comment:
you're destructuring from `this.$store.state`, but I think you meant to use `mapState` for these

Reply:
Nope, read the function.

Reply:
I don't understand this. A little bit more explanation would be nice.